Automattic/sensei
# Fix plugin zip missing block.json files

Fix the new `.block.json` files not being packaged by `composer archive`.

### Changes proposed in this Pull Request

* Update asset paths in composer config to match json files outside `assets/blocks`

### Testing instructions

* Build the plugin or download it from the Plugin Build check below
* Install it on a site and open a lesson in learning mode
* Check that the blocks are rendered
